pkg/controller/podgc/gc_controller_test.go

```diff
@@ -49,14 +49,14 @@ func NewFromClient(kubeClient clientset.Interface, terminatedPodThreshold int) (
 }
 
 func compareStringSetToList(set sets.String, list []string) bool {
+	if len(list) != len(set) {
```

Comment:
FWIW there's a subtle bug here - If the list contains multiple identical elements, then it would return false potentially incorrectly. E.g if the set was `{ "foo" }` and the list was `[]string{"foo", "foo"}`.

That's the existing behavior tho, and in the context of the tests that use it, it's ~fine.

Reply:
I have revised it. PTAL.